### ISSUE TITLE
Input (Press Quit Twice) is ignored (Quits on first press of Escape key) Latest Nightly Build

### ISSUE THREAD

**Issue by LiveFreeDead** [libretro/RetroArch#8991]
# First and foremost consider this:
- Only RetroArch bugs should be filed here. Not core bugs or game bugs
- This is not a forum or a help section, this is strictly developer oriented

## Description

Input (Press Quit Twice) is ignored, Retroarch Quits on first press of assigned key/buttons (Escape key)

### Expected behavior

In previous versions (last months nightly) it would say "Press Key Again to Quit" and would time out after 1 second if you didn't, this allowed me to set the Start Button to quit Retroarch but also use it to start games that require you press Start. 

### Actual behavior

Currently just quits instantly regardless of what the toggle is set to.

### Steps to reproduce the bug

1. Start Emulation of any ROM
2. Set Menu Item Settings/Input "Press Twice to Quit" toggled on.
3. Press Quit key/button once, will quit retroarch.

### Bisect Results

The Stable branch worked as expected, The 2019-04-28 through 2019-06-19 all fail.

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [version/commit]

### Environment information

- OS: Windows 10 x64
- Compiler: N/A

**Comment by twinaphex** [comment]
Weird - it still works for me.

**Comment by LiveFreeDead** [comment]
It must be caused by a conflict setting on my PC. because I reverted back to Stable 1.7.7 and it also quits on first press, I'll reset it all and see if I can solve it, will post any findings, but this is looking like a single PC glitch, not a bug.

**Comment by LiveFreeDead** [comment]
Corrected/Solved I guess
Step1:
I reverted my config, it still failed.

Step 2:
So I started a fresh Retroarch with v1.7.7 Stable and it still failed.

Step 3:
I rebooted my PC and ran the fresh folder and it worked
I then tried to run to original (Latest Nightly) and that also worked

I have no idea why it needed a reboot, it was very weird, not sure why it didn't start happening until I ran the new Build. Sorry to waste peoples time.
